lib/cocoapods/command/lib/create.rb

```diff
@@ -75,7 +75,7 @@ def configure_template
           UI.section("Configuring #{@name} template.") do
             Dir.chdir(@name) do
               if File.exist?('configure')
-                system({ 'COCOAPODS_VERSION' => Pod::Version }, './configure', @name, *@additional_args)
+                system({ 'COCOAPODS_VERSION' => Pod::VERSION }, './configure', @name, *@additional_args)
               else
```

Comment:
There is a spec for this - https://github.com/CocoaPods/CocoaPods/blob/master/spec/functional/command/lib/create_spec.rb#L37